strongbox-web-core/src/main/java/org/carlspring/strongbox/forms/PrivilegeForm.java

```diff
@@ -0,0 +1,46 @@
+package org.carlspring.strongbox.forms;
+
+import javax.validation.constraints.NotEmpty;
+
+/**
+ * @author Pablo Tirado
+ */
+public class PrivilegeForm
+{
+
+    @NotEmpty(message = "This field must not be empty.")
```

Comment:
Cosmetic: This error message is not very clear... (I don't yet know how it will be visualized some day, but I would say), it should be something like `A name must be specified.`. What do you think?

Reply:
I agree. It seems good to me.